geode-core/src/main/java/org/apache/geode/cache/configuration/CacheConfig.java

```diff
@@ -1140,6 +1140,34 @@ public RegionConfig findRegionConfiguration(String regionPath) {
     protected String orderPolicy;
     @XmlAttribute(name = "forward-expiration-destroy")
     protected Boolean forwardExpirationDestroy;
+    @XmlAttribute(name = "pause-event-processing")
+    protected Boolean pauseEventProcessing;
+
+    /**
+     * Gets the value of whether the processing of the events queued is paused or not
+     *
+     *
+     * @return {@link Boolean} - true if queue will be created with paused processing of the events
+     *         queued
+     *         - false if queue will be created without pausing the processing of the events queued
+     *
+     */
+    public Boolean isPauseEventProcessing() {
+      return pauseEventProcessing;
+    }
+
+    /**
+     * Sets the value of whether the processing of the events queued is paused or not
```

Comment:
Something about the creation?  the return value describes creation so maybe this should too?

Reply:
Done